### DISCUSSION TITLE
Using wavy sliders?

### DISCUSSION THREAD

**Discussion by mahozad** [FossifyOrg/General-Discussion#576] [Ideas]
Hi,

I'm doing a self-promotion here :)
Please ignore/close it if you are not interested.

Would you like to use the [wavy-slider](https://github.com/mahozad/wavy-slider) in your app?
I think Android 13 uses this kind of slider for its media controls.

[Demo](https://mahozad.ir/wavy-slider).

Although it's for Jetpack Compose, there are ways to use Composables in Views and vice versa.

**Comment by naveensingh** [comment]
> Would you like to use the [wavy-slider](https://github.com/mahozad/wavy-slider) in your app?

No :)
